### ISSUE TITLE
A3 Failing update a declaration with ipv6 virtualAddresses

### ISSUE THREAD

**Issue by notandy** [F5Networks/f5-appsvcs-extension#546]
### Environment
 * Application Services Version: 3.32.1
 * BIG-IP Version: BIG-IP 15.1.3 Build 0.21.11 Engineering Hotfix
--

<br class="Apple-interchange-newline">

### Summary
Although the inital declaration with a valid ipv6 virtualAddress suceededs, any update to the existing declaration fails with `The Service Address virtualAddress property cannot be modified. Please delete /net_2468d563_1bef_4063_aec8_8650754ab71c/2001:db8::28%2013 and recreate it` - althought the virtuaAddress is never changed in the declaration.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "action": "deploy",
    "class": "AS3",
    "declaration": {
        "class": "ADC",
        "id": "urn:uuid:1d9c2002-9be0-4bfe-9d33-0cc7eaabe2c4",
        "label": "F5 BigIP Octavia Provider",
        "net_2468d563_1bef_4063_aec8_8650754ab71c": {
            "class": "Tenant",
            "defaultRouteDomain": 2013,
            "label": "project_1203ed96bbff4a9e8bf55b52843682d9",
            "lb_05d2550f-5aba-4eab-9a2f-df575627f9f6": {
                "class": "Application",
                "label": "05d2550f-5aba-4eab-9a2f-df575627f9f6",
                "listener_420aba4b-08a7-4085-8332-6f1cce00522d": {
                    "class": "Service_L4",
                    "iRules": [],
                    "label": "ipv6 listener",
                    "persistenceMethods": [],
                    "policyEndpoint": [],
                    "virtualAddresses": [
                        "2001:db8::28"
                    ],
                    "virtualPort": 8080
                },
                "template": "generic"
            }
        },
        "schemaVersion": "3.19.0",
        "updateMode": "selective"
    },
    "historyLimit": 2,
    "logLevel": "debug",
    "persist": false,
    "trace": true
}
```
this should succeed.

2. Now post the same declaration (no changes) and observe the following error response:
```json
        {
            "code": 422,
            "message": "declaration failed",
            "response": "The Service Address virtualAddress property cannot be modified. Please delete /net_2468d563_1bef_4063_aec8_8650754ab71c/2001:db8::28%2013 and recreate it.",
            "host": "localhost",
            "tenant": "net_2468d563_1bef_4063_aec8_8650754ab71c",
            "runTime": 931
        }
```

### Expected Behavior
Same declaration should suceed and cause no changes.

### Actual Behavior
The error is emitted. Also, I found no issues in the logs. Adding control { trace: true } causes a Null-exception response from as3 and following log in the `restjavad.0.log`:

```
[WARNING][27446][13 Jan 2022 14:08:39 UTC][RestOperation] Unable to generate error body for POST http://localhost:8100/mgmt/shared/appsvcs/declare/net_2468d563_1bef_4063_aec8_8650754ab71c 422: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 2049 path $.traces.net_2468d563_1bef_4063_aec8_8650754ab71cDesired./net_2468d563_1bef_4063_aec8_8650754ab71c/Service_Address-2001:db8::28%2013.properties.arp
```

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2963.

**Comment by leondelyon3** [comment]
Hi,
we've just got the same issue with ISP customer: Orange. 
Problem seems to be related to RD different than 0. 
I've tested without RD or with RD=0 and it works fine.
Any idea when a fix will be available ?
Thanks.

**Comment by dstokesf5** [comment]
With the next release (3.36.0) already packed, a fix for this will likely be available in 3.37.0.

**Comment by LaurentGravier** [comment]
I do not see this fix in the release notes of 3.37.0. Can you please confirm if 3.38.0 will contain a fix ?

**Comment by dstokesf5** [comment]
It looks like the fix is done and ready  for the 3.38.0 release.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.38.0 release of AS3.
